library.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "STM32Ethernet",
+  "keywords": "Ethernet",
+  "description": "Arduino library to support Ethernet for STM32 based board.",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/stm32duino/STM32Ethernet"
+  },
+  "version": "1.0.4",
+  "frameworks": "arduino",
+  "platforms": "ststm32"
+}
```

Please add:
```suggestion
    "build": {
        "libArchive": false
    }
}
```
This was discussed here with @ivankravets:
https://github.com/stm32duino/STM32FreeRTOS/pull/18/files#r282812468
as a `weak` attribute exists in this lib:
https://github.com/stm32duino/STM32Ethernet/blob/27b579a9a4bf7e7ece58014d9b4f829c1a444060/src/utility/ethernetif.c#L598